jsapp/js/account/routes.constants.ts

```diff
@@ -14,7 +14,7 @@ export const AddOnsRoute = React.lazy(
   () => import(/* webpackPrefetch: true */ './add-ons/addOns.component')
 );
 export const AccountSettings = React.lazy(
-  () => import(/* webpackPrefetch: true */ './accountSettingsRoute')
+  () => import(/* webpackPrefetch: true */ './accountSettings.component')
```

Comment:
@Akuukis what's our current idea for naming these?

Reply:
Currently no idea, just stick to whatever is more consistent, simpler or faster. It's true that I'm about to suggest different naming in Linter Rules, BUT it's not there yet, so let's worry about it later. And I will try to get it done sooner.

Reply:
I'm reverting this for now. If needed I can rename this in another PR, or else, like @magicznyleszek mentioned, it's too difficult to review a whole new file